test/machine_with_event_attributes_on_save_test.rb

```diff
@@ -176,9 +176,9 @@ def test_should_allow_additional_transitions_to_previous_state_in_after_transiti
     assert_equal 'first_gear', @record.state
   end
 
-  def test_should_return_nil_on_manual_rollback
+  def test_should_return_false_on_before_transition_rollback
     @machine.before_transition { raise ActiveRecord::Rollback }
 
-    assert_equal nil, @record.save
+    assert_equal false, @record.save
```

Comment:
This line is questionable based on @mcgain's comment about activerecord default behavior.